### PR TITLE
Fix Transmission in NETCORE to handle partial success (206) response from backend

### PR DESCRIPTION
**Pull request by cijothomas**
Fix Issue #1047 

This does NOT address the other known issue channel:
https://github.com/Microsoft/ApplicationInsights-dotnet/issues/1049

- [ ] I ran [Unit Tests](https://github.com/Microsoft/ApplicationInsights-dotnet/blob/develop/.github/CONTRIBUTING.md) locally.

For significant contributions please make sure you have completed the following items:

- [ ] Design discussion issue #
- [ ] Changes in public surface reviewed
- [ ] CHANGELOG.md updated with one line description of the fix, and a link to the original issue.
- [ ] The PR will trigger build, unit test, and functional tests automatically. If your PR was submitted from fork - mention one of committers to initiate the build for you.

### CODE REVIEW

File: Test/TestFramework/Shared/StubHttpClient.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
```
Comment (cijothomas) [inline Test/TestFramework/Shared/StubHttpClient.cs]: will delete this as I am using handlers on httpclient to achieve mock.

File: src/Microsoft.ApplicationInsights/Channel/Transmission.cs [hunk 2]
```diff
@@ -158,35 +162,31 @@ public virtual async Task<HttpWebResponseWrapper> SendAsync()
...
-#if NETSTANDARD1_3
                 using (MemoryStream contentStream = new MemoryStream(this.Content))
                 {
                     HttpRequestMessage request = this.CreateRequestMessage(this.EndpointAddress, contentStream);
+                    HttpWebResponseWrapper wrapper = null;
 
                     using (var ct = new CancellationTokenSource(this.Timeout))
                     {
-                        await client.SendAsync(request, ct.Token).ConfigureAwait(false);
+                        var response = await client.SendAsync(request, ct.Token).ConfigureAwait(false);
```
Comment (lmolkova) [inline src/Microsoft.ApplicationInsights/Channel/Transmission.cs]: HttpResponseMessage is disposable, please add using block

File: Test/Microsoft.ApplicationInsights.Test/Shared/Channel/TransmissionTest.cs [hunk 3]
```diff
@@ -94,207 +100,197 @@ public void SetsTimeoutToSpecifiedValue()
...
-                finishBeginGetRequestStream.Set();
+                    // ACT
+                    HttpWebResponseWrapper result = await transmission.SendAsync();
+                }
             }
-            
+
             [TestMethod]
-            public void DoesNotAbortRequestThatWasSentSuccessfully()
+            [Ignore("https://github.com/Microsoft/ApplicationInsights-dotnet/issues/1047")]
```
Comment (Dmitry-Matveev) [inline Test/Microsoft.ApplicationInsights.Test/Shared/Channel/TransmissionTest.cs]: Minor: Was it a typo, #1049 sounds like an issue about timeout?

File: Test/Microsoft.ApplicationInsights.Test/Shared/Channel/TransmissionTest.cs [hunk 4]
```diff
@@ -1,6 +1,13 @@
-﻿namespace Microsoft.ApplicationInsights.Channel
+﻿using System.Linq;
```
Comment (Dmitry-Matveev) [inline Test/Microsoft.ApplicationInsights.Test/Shared/Channel/TransmissionTest.cs]: Minor: "Using" jumped out of the namespace.

**Pull request merged**